src/frame/vulkan.rs

```diff
@@ -292,16 +310,22 @@ impl Vulkan {
             .usage(vk::ImageUsageFlags::TRANSFER_SRC)
             .sharing_mode(vk::SharingMode::EXCLUSIVE);
 
-        let frame_image = unsafe { self.device.create_image(&frame_image_create_info, None)? };
+        let frame_image = unsafe {
+            self.device
+                .create_image(&frame_image_create_info, None)
+                .map_err(anyhow::Error::msg)?
+        };
 
+        // Memory requirements info
         let frame_image_memory_req_info =
-            vk::ImageMemoryRequirementsInfo2::builder().image(frame_image);
+            vk::ImageMemoryRequirementsInfo2::default().image(frame_image);
 
-        // Prepare the structures to get memory requirements into, then get the requirements
+        // Memory requirements
```

Comment:
```suggestion
        // Prepare the structures to get memory requirements into, then get the memory requirements
```

Also a good place to preserve the comment :)